_guides/007-gethelp.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Get help
+anchor: get_help
```

Comment:
Left nav is a bit weird with:

> Get help
     - Get help

Not sure the extra level needed?